bedrock/settings/base.py

```diff
@@ -488,7 +487,7 @@ def lazy_langs():
 
 def set_whitenoise_headers(headers, path, url):
     if "/fonts/" in url or "/caldata/" in url:
-        cache_control = "public, max-age={}".format(604800)  # one week
+        cache_control = f"public, max-age={604800}"  # one week
```

Comment:
Why not just skip the string building? Not sure why it was there in the first place.

```suggestion
        cache_control = f"public, max-age=604800"  # one week
```

Reply:
Nice catch